.gitignore

```diff
@@ -250,23 +250,18 @@ paket-files/
 .idea/
 *.sln.iml
 
-# Visual Studio Code
-.vscode/launch.json
-.vscode/tasks.json
-
-# launchSettings.json
-launchSettings.json
-
 # Node and .NET Core runtime images dockerfiles, that are auto generated
 images/runtime/node/*/Dockerfile
 images/runtime/dotnetcore/*.*/
 images/runtime/python/*.*/
 images/runtime/php/*.*/
 
-# Go packages, if Oryx is added to GOPATH (which it should)
-pkg/
-
 # Go packages from GitHub
-src/github.com/
-src/startupscriptgenerator/vendor/
-src/golang.org/
\ No newline at end of file
+src/startupscriptgenerator/bin/
+src/startupscriptgenerator/src/github.com/
```

Comment:
is this still needed even after moving every external packages to the `vendor` folder?

Reply:
yes, because we do a `go get` for the `dep` tool itself which puts its stuff in this `github.com` folder

Reply:
maybe we better document this in the .gitignore file, to help prevent future confusion

Reply:
Sure, I agree. Will add a comment in my next PR related to making dotnetcore startup script use go